runtime/doc/gui_w32.txt

```diff
@@ -1,262 +1,14 @@
-(for unclear reasons...), thus the "Edit with Vim" menu entry is still useful.
-
-One reason to add this is to be able to edit HTML files directly from Internet
-Explorer.  To enable this use the "Tools" menu, "Internet Options..." entry.
-In the dialog select the "Programs" tab and select Vim in the "HTML editor"
-choice.  If it's not there than installing didn't work properly.
-
-Doing this manually can be done with this script:
-
-----------------------------------------------------------
-REGEDIT4
-
-[HKEY_CLASSES_ROOT\Applications\gvim.exe]
-
-[HKEY_CLASSES_ROOT\Applications\gvim.exe\shell]
-
-[HKEY_CLASSES_ROOT\Applications\gvim.exe\shell\edit]
-
-[HKEY_CLASSES_ROOT\Applications\gvim.exe\shell\edit\command]
-@="c:\\vim\\vim62\\gvim.exe \"%1\""
-
-[HKEY_CLASSES_ROOT\.htm\OpenWithList\gvim.exe]
-
-[HKEY_CLASSES_ROOT\*\OpenWithList\gvim.exe]
-
-----------------------------------------------------------
-
-Change the "c:\\vim\\vim62" bit to where gvim.exe is actually located.
-
-To uninstall this run the Vim uninstall program or manually delete the
-registry entries with "regedit".
-
-==============================================================================
-3. Using the clipboard					*gui-clipboard*
-
-Windows has a clipboard, where you can copy text to, and paste text from.  Vim
-supports this in several ways.  For other systems see |gui-selections|.
-
-The "* register reflects the contents of the clipboard.  |quotestar|
-
-When the "unnamed" string is included in the 'clipboard' option, the unnamed
-register is the same.  Thus you can yank to and paste from the clipboard
-without prepending "* to commands.
-
-The 'a' flag in 'guioptions' is not included by default.  This means that text
-is only put on the clipboard when an operation is performed on it.  Just
-Visually selecting text doesn't put it on the clipboard.  When the 'a' flag is
-included, the text is copied to the clipboard even when it is not operated
-upon.
-
-							*mswin.vim*
-To use the standard MS-Windows way of CTRL-X, CTRL-C and CTRL-V, use the
-$VIMRUNTIME/mswin.vim script.  You could add this line to your _vimrc file: >
-	source $VIMRUNTIME/mswin.vim
-
-Since CTRL-C is used to copy the text to the clipboard, it can't be used to
-cancel an operation.  Use CTRL-Break for that.
-
-CTRL-Z is used for undo.  This means you can't suspend Vim with this key, use
-|:suspend| instead (if it's supported at all).
-
-						*CTRL-V-alternative* *CTRL-Q*
-Since CTRL-V is used to paste, you can't use it to start a blockwise Visual
-selection.  You can use CTRL-Q instead.  You can also use CTRL-Q in Insert
-mode and Command-line mode to get the old meaning of CTRL-V.  But CTRL-Q
-doesn't work for terminals when it's used for control flow.
-
-NOTE: The clipboard support still has a number of bugs.
+Vim for Win32
 
 ==============================================================================
 4. Shell Commands					*gui-shell-win32*
```

Comment:
Should the section numbers be removed since you got rid of the table of contents?